src/main/java/ca/uhn/fhir/jpa/starter/CustomServerCapabilityStatementProviderR4.java

```diff
@@ -0,0 +1,66 @@
+package ca.uhn.fhir.jpa.starter;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.annotation.Nonnull;
+import javax.servlet.http.HttpServletRequest;
+
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.CapabilityStatement;
+import org.hl7.fhir.r4.model.CapabilityStatement.CapabilityStatementRestSecurityComponent;
+import org.hl7.fhir.r4.model.Extension;
+import org.hl7.fhir.r4.model.Meta;
+import org.hl7.fhir.r4.model.UriType;
+import org.springframework.context.annotation.Configuration;
+
+import com.google.gson.Gson;
+
+import ca.uhn.fhir.jpa.api.config.DaoConfig;
+import ca.uhn.fhir.jpa.api.dao.IFhirSystemDao;
+import ca.uhn.fhir.jpa.provider.r4.JpaConformanceProviderR4;
+import ca.uhn.fhir.jpa.searchparam.registry.ISearchParamRegistry;
+import ca.uhn.fhir.rest.api.server.RequestDetails;
+import ca.uhn.fhir.rest.server.RestfulServer;
+
+@Configuration
+public class CustomServerCapabilityStatementProviderR4 extends JpaConformanceProviderR4 {
+
+	private static final String EXTENSION_MAP = System.getenv("OAUTH_CAPABILITY_EXTENSION");
```

Comment:
I'd rather not have a map specified in an environment variable. Consider how difficult, and error-prone, it would be for users to specify that.

Let's go with env vars like:
```
OAUTH_TOKEN_URL
OAUTH_MANAGE_URL
```

Reply:
okay. I will update PR accordingly